### DISCUSSION TITLE
Flit clever-f Style Repeat Configuration

### DISCUSSION THREAD

**Discussion by JaredLabs** [LazyVim/LazyVim#980] [Q&A] [answered]
When using the `f` operator that is configured by flit.nvim I am having trouble understanding how to reach the second occurrence of a target character. 

For example,
![image](https://github.com/LazyVim/LazyVim/assets/137745027/bdabf899-cc69-4726-acc9-b117b2a5462e)

How would I go about reaching the light blue label for `L` (i.e. the second `L` label) I had assumed I could just press `f` again, but that doesn't seem to be working. I have made very few modifications to the overall configuration. Rather new to neovim so any help or guidance would be appreciated.

**Comment by jyuan0** [comment, marked as answer]
Type `<space>` and then the target label (`L`).

flit.nvim doesn't really have any docs, and it's building upon leap.nvim, so your best bet for docs is there:
> The blue labels indicate the "secondary" group of matches, where we start to reuse the available labels for a given pair (s, f, n... again). You can reach those by prefixing the label with \<space>, that switches to the subsequent match group. For example, to jump to the "blue" j target, you should now press r\<space>j.

https://github.com/ggandor/leap.nvim#usage

**Reply by JaredLabs** [reply]
Tremendous! Thank you, I noticed the docs were lacking a bit, but I see now that its somewhat of a superset of leap. I will look there for any follow-ups.

**Comment by folke** [comment]
You could also try out the new flash plugin instead https://www.lazyvim.org/plugins/extras/editor.flash
